traceview/discovery.py

```diff
@@ -17,9 +17,7 @@ class Action(Resource):
     PATH = "actions"
 
 
-class App(Resource):
-
-    PATH = "apps"
+# for App class, see app.py
```

Comment:
In the docs PR (I'll hipchat you) it's listed under both Discovery and App Management.  Per comment in this PR intro, it isn't ideal to have App functionality split into these two silos in the API, docs, or client, but that's a bigger change.

One alternative I considered is leaving `App` class in discovery.py, and adding a new `AppManagement` class in app.py.  Could change to that, though this felt similar to what was done with `Host` stuff.